packages/theme-generator/src/compile-css.js

```diff
@@ -28,3 +28,24 @@ export default function (themeFilePath) {
 
   return result.css;
 }
+
+export function compileBase() {
+  const fullTheme = `@import "main";`;
+
+  const dirname = fileURLToPath(new URL('.', import.meta.url));
```

Comment:
Assuming Node.js `^20.11.0`:
```suggestion
  const dirname = import.meta.dirname;
```